editor.planx.uk/src/@planx/components/MapAndLabel/Editor.tsx

```diff
@@ -0,0 +1,28 @@
+import { ComponentType as TYPES } from "@opensystemslab/planx-core/types";
+import { useFormik } from "formik";
+import React from "react";
+
+import { EditorProps } from "../ui";
```

Comment:
Tried to get this working with the path "@planx/..." but it kept throwing an error so I changed to a relative path